docs/projects/cv_zebrafish/about.md

```diff
@@ -0,0 +1,63 @@
+---
+id: about
+title: CV Zebrafish
+custom_edit_url: null
+---
+
+<!-- Optional header image -->
+<!-- ![Header Alt Text](header.png) -->
+
+## Overview
+
+The CV Zebrafish project aims to develop an interactive, user-friendly software tool to track and analyze zebrafish body parts and movements from high-resolution, high-speed imaging data. It will integrate with machine learning outputs like DeepLabCut to process large datasets and generate detailed metrics, including speed, fin angles, movement timing, phase synchronization, and behavior classification. The modular design will make it adaptable for researchers at SLU and beyond, supporting diverse movement analysis and visualization needs in aquatic behavioral studies.
+
+### Information
+
+- **Source Code:** [https://github.com/oss-slu/cv_zebrafish](https://github.com/oss-slu/cv_zebrafish) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/cv_zebrafish)
+- **Client:** Mohini Sengupta, Ph.D.
+- **Current Tech Lead:** Shahid Malik [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Shahid1Malik)  
+- **Developers:**
+  - [Developer 1 Name] [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/USERNAME) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/USERNAME/)
```

Comment:
I like that you're thinking ahead. Let's comment these out until we have team members identified on/around August 25.